src/lib/http.ts

```
@@ -0,0 +1,127 @@
+
+class SessionToken {
+  private token = "";
+
+  // Getter
+  get value() {
+    return this.token;
+  }
+
+  // Setter
+  set value(token: string) {
+    if (typeof window === undefined) {
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)